sse/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"> <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.javaee8</groupId>
+        <artifactId>samples-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>sse</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Java EE 8 Samples: Server Sent Events</name>
+
+    <modules>
+        <module>producer</module>
+    </modules>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.javaee8</groupId>
+            <artifactId>test-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-sse</artifactId>
+            <version>2.26</version>
```

Comment:
org.glassfish.jersey.media dependency can't be here, this is specific for GlassFish ad Payara, but the tests are intended to run on any server